main/createUiDefinition.json

```diff
@@ -34,18 +34,75 @@
         },
         "visible": true
       },
+      {
+        "name": "vmSize",
+        "type": "Microsoft.Compute.SizeSelector",
+        "label": "VM size",
+        "toolTip": "Select the size of VM.",
+        "recommendedSizes": [
+          "Standard_F4s"
```

Comment:
I think recommended size needs to be "Standard_F8s"

Reply:
Done